Nos/Assets/Localization/ImagePicker.xcstrings

```diff
@@ -16,6 +16,12 @@
             "value" : "Camera"
           }
         },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Cámera"
```

Comment:
```suggestion
            "value" : "Cámara"
```

Reply:
@joshuatbrown I don't understand or see what you changed here.